frontend/src/components/Header/header.module.scss

```diff
@@ -5,9 +5,17 @@
   background-color: $colour-dark-blue;
   display: grid;
   grid-template-areas: 'left middle right';
-  grid-template-columns: 2.5rem 1fr 100px;
+  grid-template-columns: 2.5rem 1fr;
 }
 
 .headerRight {
+  display: flex;
   grid-area: right;
 }
+
+.userName {
```

Comment:
This might not be the best way to do this - @tjanowsk any ideas on how to get the username text to be centered in the header?

Reply:
I seem to recall that setting left and right margins to auto can be used to center the text.
Alternatively, can try adding `align-content` and `justify-content` to `.headerRight`
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container

Reply:
Yeah I tried `align-items`, and it moved the login button down level with the text. I'll have another look.